package.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "@aave/core-v3": "^1.19.3",
     "@aave/deploy-v3": "^1.56.2",
-    "@coinbase/coinbase-sdk": "^0.8.0",
+    "@coinbase/coinbase-sdk": "^0.9.0",
```

Comment:
probably doesn't matter to be 0.9.1 for that type issue, but can we bump to keep versions up to date in any case?